### DISCUSSION TITLE
How to fix icons in the new rofi update ( Iggy 2024 )?

### DISCUSSION THREAD

**Discussion by CTZxVULKAN** [davatorium/rofi#1386] [Theming] [answered]
My rofi theme was using a pre-1.6.1 rasi theme. The theme was broken with the new update.
Adding the following from the [README](https://github.com/davatorium/rofi/releases/tag/1.7.0) fixed it somewhat however the icons still look bad.
```
element-text {
    background-color: inherit;
    text-color:       inherit;
}
``` 
Icons **before** update :

![rofi-old](https://user-images.githubusercontent.com/65299153/131283509-05d21f72-7f7f-442a-bb86-29170c00989f.png)

Icons **after** update :

![rofi-updated](https://user-images.githubusercontent.com/65299153/131283546-8a7739bc-35e4-4f62-b54e-cee591396f27.png)

I am using this for running rofi

```
rofi -display-drun "" -modi drun -show drun -font "Hack 20" -icon-theme "Tela" -show-icons
```

How do i make my icons look like the pre update theme ?

**Comment by TonCherAmi** [comment, marked as answer]
It's hard to tell for certain without seeing your config, but I think you're missing `element-icon` styling, try something like this:

```css
element-icon {
  size: 32;
  margin: 0 10 0 0;

  background-color: inherit;
  text-color:       inherit;

  vertical-align: 0.5;
}
```

You might need to tweak `size`, `margin`, and `vertical-align` a bit so that it fits your theme better.

If that doesn't work, please provide your full config so that the issue can be diagnosed properly.

**Reply by CTZxVULKAN** [reply]
Yes this works fine . 
Thanks :)

**Reply by DaveDavenport** [reply]
What version did you run before update?  this was a change in 1.6.0 (it was mentioned there in the releasenotes).
